sspl_test/alerts/os/test_service_monitor_sensor.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+# -*- coding: utf-8 -*-
+
+import time
+
+from default import world
+from rabbitmq.rabbitmq_ingress_processor_tests import RabbitMQingressProcessorTests
+from rabbitmq.rabbitmq_egress_processor import RabbitMQegressProcessor
+from common import check_sspl_ll_is_running
+from cortx.utils.service import DbusServiceHandler
+
+RESOURCE_TYPE = "node:sw:os:service"
+WAIT_TIME = 75   # 60s wait_time + 10s buffer
```

Comment:
I realize now that this buffer time should be equal to the `polling_frequency - 1`. 
so total time required for this test will be :
inactive/fault alert :        90s    (60s wait_time + 30s current polling_frequency)
fault_resolved alert:     +  5s
restart no alert test:     + 90s
-------------------------------------
total time               :       185s
**Toatal time will increase with increase in polling time..**

Reply:
sure.. (y)